spotlight-client/src/contentModels/types.ts

```diff
@@ -90,4 +89,4 @@ export type SystemNarrativeMapping = {
   [key in SystemNarrativeTypeId]?: SystemNarrative;
 };
 
-export type Narrative = SystemNarrative | RacialDisparitiesNarrative;
```

Comment:
per comments about the links above, I don't think we are going to want to make this change in the end.